src/components/pages/RoleSignup/Applications/Mentee.js

```diff
@@ -47,14 +48,18 @@ const initialFormValues = {
 const Mentee = () => {
   const [formValues, setFormValues] = useState(initialFormValues);
 
+  let history = useHistory();
+
   const postNewAccount = async newAccount => {
     try {
       const response = await axios.post(
         `${process.env.REACT_APP_API_URI}/application/new/mentee`,
         newAccount
       );
+      history.push('/apply/success');
       console.log('post response', response);
```

Comment:
Could you delete this `console.log`? This must have been left in from a previous PR.

Reply:
Now deleted.